backend/docker/dev/docker-compose.yml

```
@@ -122,6 +122,36 @@ services:
+    environment:
+      - TZ=Asia/Seoul
+    deploy:
+      resources:
+        limits:
+          cpus: '0.1'
+          memory: 128M
+        reservations:
+          cpus: '0.05'
+          memory: 64M
+    healthcheck:
+      test: [ "CMD", "wget", "--spider", "-q", "http://localhost:9080/ready" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)